pkg/products/fuse/reconciler.go

```diff
@@ -319,3 +322,19 @@ func (r *Reconciler) reconcileCustomResource(ctx context.Context, installation *
 	// if there are no errors, the phase is complete
 	return integreatlyv1alpha1.PhaseCompleted, nil
 }
+
+func preUpgradeBackupExecutor(installation *integreatlyv1alpha1.RHMI) backup.BackupExecutor {
+	return backup.NewNoopBackupExecutor()
+
+	// Holding off until Fuse 7.6 GA, that allows for external databases:
```

Comment:
Lets add a note to the fuse 7.6 jira, or make a new jira around this code, so it doesn't get forgotten, please.